ProtocolHandler.pm

```diff
@@ -207,15 +214,16 @@ sub parseDirectHeaders {
 
 	my $song = $client->streamingSong();
 	$ct =~ s/(?:m4a|mp4)/aac/i;
+	$ct = Slim::Music::Info::mimeToType($ct);
```

Comment:
What is this part about? Necessary? Tested? What problem does it solve?

Reply:
I've seen that the expected contenType result of parseDirectHeaders is a not an audio/xxx but just the xxx part and that is used as a key of the player's supported formats. See line 549-690 of Squeezebox2.pm. Without that, when returned audio/x-flac (for example, this is what RP gives), then playback fails as format is claimed to be not recognized by player. This is also what HTTP.pm does